src/Index/utils.js

```diff
@@ -19,6 +19,66 @@ const updateRedux = async projectNodeCounts => getReduxStore().then(
   },
 );
 
+// Chunk into groups of 15
+// Chunking is necessary to avoid hitting Postgres limits
+export const getChunkedPeregrineRequestUrls = (nodesToRequest) => {
+  const requestUrls = [];
+
+  let k = 0;
+  const chunkSize = 15;
```

Comment:
this `chunkSize` should be configuable but optional from config file, just like `lineLimit`. This 15 can be the default value if the it is omit

If an env have much more projects we may need to decrease this number further